### PR TITLE
DBZ-3024 Migrate DebeziumContainer enhancements for DBZ-2950 and DBZ-2952 into main repository

### PR DESCRIPTION
**Pull request by rk3rn3r**
DBZ-2950 and DBZ-3002 need some enhancements to DebeziumContainer. In order to have SNAPSHOT packages in the Sonatype repository and new code in nightly builds of debezium/connect containers I need to have this migrated to the main repo in order to continue with DBZ-2950 (add MySQL connector to UI backend) prior finishing DBZ-3002 (re-add Postgress replication slot validation).

closes https://issues.redhat.com/browse/DBZ-3024

### CODE REVIEW

**Comment by rk3rn3r** [comment]
@gunnarmorling we agreed on that DBZ-3002 (re-add proper Postgres replication slot validation) will continue only in "spare time"/with low prio, but it holds some improvements to DebeziumContainer that I need for DBZ-2950 (add MySQL connector to DBZ UI). I need this change as SNAPSHOT in the Sonartype repository and included in the `debezium/connect:nightly` container, that's I want to have it merged prior finishing DBZ-3002.

Basically it's a fix for the latest changes on DebeziumContainer where the constructor was made private but not everyone is using the `connect` container in the "latest" version, plus some additional functionality that I migrate out of DBZ-3002 here.

File: debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/DebeziumContainer.java [hunk 1]
```diff
@@ -110,25 +119,49 @@ public String getPauseConnectorURI(String connectorName) {
         return getConnectorURI(connectorName) + "/pause";
     }
 
+    /**
+     * Returns the "/connectors/<connector>/pause" endpoint.
+     */
+    public String getResumeConnectorURI(String connectorName) {
```
Comment (gunnarmorling) [inline debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/DebeziumContainer.java:125]:
```suggestion
    public String getResumeConnectorUri(String connectorName) {
```
(It's the style we generally use)

**Review by gunnarmorling — COMMENTED** [review]
LGTM; one tiny comment inline, and could you fix the typo in the commit message? Thx!

File: debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/DebeziumContainer.java (hunk 1 above)
Comment (rk3rn3r) [reply debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/DebeziumContainer.java:125]: Hmmm... all other methods have a similar name. Renaming all?

**Review by gunnarmorling — APPROVED** [review]
LGTM, awaiting CI. Thx!
